database/charts/templates/database.yaml

```diff
@@ -0,0 +1,127 @@
+---
+kind: PersistentVolumeClaim
```

Comment:
can we do the split of files now or is that for later, I like the pattern , when we have specific file for specific thing
ex:-secret.yml, deployment.yml, imagestream.yml ..., follows the helm patterns

Reply:
It's a good pattern, but there's too much object overlap between database and database-backup.  I'll handle the split when we revise the chart structure (probably subcharts).